integration-cli/docker_cli_volume_test.go

```diff
@@ -212,7 +212,7 @@ func (s *DockerSuite) TestVolumeCliRm(c *check.C) {
 func (s *DockerSuite) TestVolumeCliNoArgs(c *check.C) {
 	out, _ := dockerCmd(c, "volume")
 	// no args should produce the cmd usage output
-	usage := "Usage:	docker volume [OPTIONS] [COMMAND]"
+	usage := "Usage:	docker volume COMMAND"
```

Comment:
Since it's only `-h` (which is basically what you're looking at if you see the `Usage`), I'm ok with keeping it like this, otherwise we end up with

```
docker volume [OPTIONS] [COMMAND] [OPTIONS]
```